packages/cli/src/commands/init.ts

```diff
@@ -130,8 +137,16 @@ function getContentFromPromptResult(result: PromptResult, useJsFile: boolean): s
     }
     const content: Config = { idl: result.idlPath, before: [], scripts };
 
-    if (!useJsFile) {
+    if (configFileType == 'json') {
         return JSON.stringify(content, null, 4);
+    } else if (configFileType == 'gill') {
+        return `import { createCodamaConfig } from "gill";\n\n` +
+            `export default createCodamaConfig({ \n\t` +
+            `idl: "${result.idlPath}", \n\t` +
+            `clientJs: "${result.jsPath}", \n` +
+            result.scripts.includes('rust')
+            ? `clientRust: "${result.rustPath}", \n`
+            : `` + `});`;
```

Comment:
Thank you for including the Rust client in there!

There's one more thing to consider. The JS client is also optional. So if the user only wants a Rust renderer, we shouldn't display the `clientJs` option on there. Is this currently supported by Gill?

Reply:
gill's assumption is that if a user is using the `createCodamaClient` function they want a js client that works with gill (since they have to have it installed). So the `clientJs` argument is required.

Reply:
Gotcha, then perhaps we shouldn't allow the user not to select the JS client when using the `--gill` option?

Reply:
yeah, I think that's a better approach. They will still need to input the js client data. I think it would be the best experience if users that passed the `--gill` flow see the same "select which client" buttons, but with the js one locked as selected/on with a note saying something to the effect of "you are using gill, so you get a js client"

Reply:
Hey @nickfrosty, is this something you'd want to tackle in this PR? Or should we just merge this as-is for now and you can add that safety guard later on for Gill?

Reply:
Sounds good! Issue created here: https://github.com/codama-idl/codama/issues/726.